### DISCUSSION TITLE
[5.0] Lucid Query With Distinct & OrderBy

### DISCUSSION THREAD

**Discussion by lsfratel** [adonisjs/core#1828] [Help] [answered]
Hello guys, I'm trying to get the latest (distinct by manga) chapter of manga oderned by created_at.

```js
//relationship:
manga -> hasMany(chapter)
chapter -> belongsTo(manga)
```

the code:
```js
const latestMangaUpdates: Array<Chapter> = await Chapter.query()
      .orderBy('created_at', 'desc')
      .distinct('manga_id')
      .limit(24)
      .preload('manga')
```
and getting this error:
`select distinct "manga_id" from "chapters" order by "created_at" desc limit $1 - for SELECT DISTINCT, ORDER BY expressions must appear in select list`

Sorry if my explanation was not clear, English is not my native language. So how would this query be using lucid?

**Comment by hariprasath-yadav** [comment, marked as answer]
use like
```
const latestMangaUpdates: Array<Chapter> = await Chapter.query()
      .select('created_at')
      .orderBy('created_at', 'desc')
      .distinct('manga_id')
      .limit(24)
      .preload('manga')
```
